components/typography.tsx

```diff
@@ -61,7 +61,6 @@ function Typography(
       ) : (
         <span
           className={
-            // eslint-disable-next-line total-functions/no-array-subscript
```

Comment:
I fixed https://github.com/danielnixon/eslint-plugin-total-functions/issues/11, so this (sort of) false positive is fixed.